README.md

```diff
@@ -1,3 +1,13 @@
+![OnLaunch](https://github.com/kula-app/OnLaunch/blob/main/public/github_header.png)
```

Comment:
The folder `public` is included when deploying the app, but the file is only used for documentation.

Move it to `docs/res`